CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+### Bugfixes
+
+#### node and oidc
+
+- The OIDC issuer profile is used to negociate the preferred signature algorithm
+for ID tokens.
+
+#### node
+
+- During client registration, the client explicitly registers both the 'refresh_token'
```

Comment:
Minor, but as a developer who just wants to be able to connect to people's Pods, I have no idea how this affects me :P How do I know if this bug affected me, and if it did, how?

Reply:
725b5c335a8a16de7ecfaa20c0235a30a85398b5